packages/react-composites/src/composites/CallWithChatComposite/MobilePane.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { concatStyleSets, DefaultButton, FontIcon, Stack } from '@fluentui/react';
+import { useTheme } from '@internal/react-components';
+import React from 'react';
+import {
+  paneBodyContainer,
+  scrollableContainer,
+  scrollableContainerContents
+} from '../common/styles/ParticipantContainer.styles';
+import { useCallWithChatCompositeStrings } from './hooks/useCallWithChatCompositeStrings';
+import {
+  hiddenMobilePaneStyle,
+  mobilePaneBackButtonStyles,
+  mobilePaneButtonStyles,
+  mobilePaneControlBarStyle,
+  mobilePaneStyle
+} from './styles/MobilePane.styles';
+
+export const MobilePane = (props: {
+  onClose: () => void;
+  onChatButtonClicked: () => void;
+  onPeopleButtonClicked: () => void;
+  children: React.ReactNode;
+  hidden: boolean;
+  dataUiId: string;
+  activeTab: MobilePaneTab;
+}): JSX.Element => {
+  // We hide the mobile pane instead of not rendering the entire pane to persist certain elements
```

Comment:
Great comment. Comments like this make our future self soooo grateful.

Reply:
+1!